### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Certificates_Screen Reader: Narrator is not reading the controls in meaningful sequence in Certificates page

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#73]
**Environment Details:**
URL: https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated:MAS1.3.2, MAS2.4.3

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Verify that screen reader is reading the contents and controls in a meaningful sequence.

**Actual Result:**
Narrator is not reading the contents and controls in meaningful sequence.
After opening Certificates page Narrator focus is going first on the Refresh button then on the search box

**Expected Result:**
Screen Reader should read the information in correct order throughout the process. Focus should land first on the Search box then it should go to the Refresh button. The order should be from left to right and top to bottom in the page.

**User Impact:**
Visually Impaired Users might face difficulty in navigating through the contents and controls if the focus order is not proper 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachments:**
[Bug_73_Narrator focus is not in meaningful sequence.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317764/Bug_73_Narrator.focus.is.not.in.meaningful.sequence.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Non standard UI, Won't fix

**Comment by jhkimnew** [comment]
duplicate bug to #63

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator; #MAS2.4.3; #Keyboard

**Comment by fsteffi** [comment]
#A11ySev3;
